src/app/components/modules/simplified-lunch/simplified-lunch.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="module-container">
+<div *ngIf="loading" class="loading-container">
+	<div mymicds-blur class="loading">
+		<h1>Fetching the Lunch Menu...</h1>
+	</div>
+</div>
+<div *ngIf="!loading" class="lunch-container container">
+
+
+	<!-- Actual Lunch --> 
+	<div class="lunch-week" routerLink='/lunch'>
+		<!-- Loop through Weekdays -->
+		<div myMicds-blur class = "lunch-day text-center" [class.today] = "">
```

Comment:
`myMICDS-blur` should actually be `mymicds-blur` like the `.loading-container` above. Also, don't put a space between html attributes, the equal sign, and the value so `class = ""` should be `class=""` and `[class.today] = ""` should be `[class.today]=""`